server/server.js

```diff
@@ -138,6 +138,32 @@ function findByPath(req, res, path) {
   );
 }
 
+function findByPathForVersionDate(req, res, path, version_date) {
+  executeQueryWithCallback(
+    `query resume($path: String_comparison_exp, $version_date: date_comparison_exp) {
+      resume(where: {path: $path, version_date: $version_date}) {
+        content
+        metadata
+        path
+        version
+        last_modified
+      }
+    }
+    `,
+    { path: { _eq: path }, version_date: { _eq: version_date} },
+    req,
+    res,
+    function(data) {
+      if (data.rows.length < 1) {
+        findByUuid(req, res, path);
```

Comment:
Why is it falling back `findByUuid`? Shouldn't we respond 404 here?

Reply:
I just used code that was already in place for `findByUuid`